packages/sit-onyx/src/components/OnyxInput/OnyxInput.vue

```diff
@@ -94,13 +99,22 @@ const { disabled, showError } = useFormContext(props);
 const skeleton = useSkeletonContext(props);
 const errorClass = useErrorClass(showError);
 useAutofocus(input, props);
-const showPassword = ref(false);
+
+const showPassword = useVModel({
+  props,
+  emit,
+  key: "showPassword",
+  default: false,
+});
+
 const displayType = computed(() => {
   if (props.type === "password" && showPassword.value) {
     return "text";
   }
   return props.type;
 });
+
+watch(showPassword, (value) => emit("update:showPassword", value));
```

Comment:
```suggestion
```

The watch is not needed since the `useVModel` already calls it :)
So basically useVModel is similar to Vues defineModel but we needed to implement a custom utility because defineModel behaves differently in edge cases